src/http/server/server.cr

```diff
@@ -92,6 +92,8 @@ class HTTP::Server
     property ssl : OpenSSL::SSL::Context?
   end
 
+  @server : TCPServer?
+  @server = nil
```

Comment:
You don't need to initialize it to nil. This is already the case.

Apart from that, this looks good to me.